docs/markdown/ilp-flow.md

```diff
@@ -0,0 +1,40 @@
+# Basic Interledger Payment Flow
+ 
+This section helps you become more familiar with Interledger by outlining the parts of an Interledger payment system and explaining the steps involved in an Interledger payment.
+ 
+## Interledger payment
+ 
+The Interledger payment system consists of four main components:
+- Sender - Initiates a value transfer.
+- Router - Applies currency exchange and forwards packets of value.
+- Receiver - Receives the value.
+- ILP Packet - ILPv4 has three packet types: Prepare, Fulfill, and Reject, which roughly correspond to request, response, and error messages.
+
+In the following scenario, the sender sends a Prepare packet to the connecting router. The routers forward the packet until it reaches the receiver. The receiver then accepts or rejects the packet by sending a Fulfill packet or a Reject packet.
+ 
+ ![ILP-packets](../images/ilp-packets.png)
+
+## Interledger payment flow
+ 
+### Prerequisites
+
+1. The sender and receiver have accounts with at least one router on the Interledger network.
+2. The sender knows the receiver's payment pointer, which is required for exchanging necessary payment information.
+ 
+### Payment flow
+
+1. The sender initiates a payment by connecting with the receiver over an out-of-band authenticated channel and obtaining the receiver's ILP address and a secret hash condition.
```

Comment:
While possible, are you actually trying to describe SPSP here? Or is this meant to be a more generic thing?

Reply:
My intention is to keep it generic. I didn't want to name SPSP or `moneyd` in this flow because similar applications can be created by anyone in the future but the general concept will remain the same (I guess?).

Reply:
(Lois here): I've taken this file over in Prachi's absence. How about persistent payment identifier then, as a payment pointer is meant to be a persistent identifier?